ts/theme/components/Header.ts

```diff
@@ -7,7 +7,17 @@ import variables from "../variables";
 export default (): Theme => {
   return {
     "NativeBase.Left": {
-      flex: 0.2
+      "NativeBase.Button": {
```

Comment:
You are changing a component used in all the App. Have you checked for side effects?

Reply:
I have checked in all app screen, there is not side effect.